### PR TITLE
SSL options c_str() fix

### PR DESCRIPTION
**Pull request by messa**
CXX-863

### CODE REVIEW

**Comment by acmorrow** [comment]
Thank you for the bug report and the pull request. Could you please

- Update the commit messages to reference the ticket (see the recent commit history for examples)
- Sign the MongoDB contributors agreement at https://www.mongodb.com/legal/contributor-agreement

And we will get this merged.

File: src/mongocxx/private/ssl.hpp [hunk 1]
```diff
@@ -26,20 +26,21 @@ namespace options {
 #if defined(MONGOC_HAVE_SSL)
 inline ::mongoc_ssl_opt_t make_ssl_opts(const ssl& ssl_opts) {
     ::mongoc_ssl_opt_t out;
+    memset(&out, 0, sizeof(out));
```
Comment (acmorrow) [inline src/mongocxx/private/ssl.hpp]: I think you can just write this as:

::mongoc_ssl_opt_t out{};

Instead of needing to call memset.

Comment (messa) [inline src/mongocxx/private/ssl.hpp]: Thanks :) Fixed in fbc13bd.

**Comment by messa** [comment]
I've already signed the MongoDB Contributor Agreement on Mar 14, with my Github username.

**Comment by acmorrow** [comment]
@messa Great, and thanks for the PR. We will get this merged.

**Pull request merged**